slack/table_slack_conversation_members.go

```diff
@@ -0,0 +1,61 @@
+package slack
+
+import (
+	"context"
+
+	"github.com/slack-go/slack"
+
+	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+func tableSlackConversationMembers() *plugin.Table {
+	return &plugin.Table{
+		Name:        "slack_conversation_members",
+		Description: "Retrieve members of a conversation.",
+		List: &plugin.ListConfig{
+			KeyColumns: plugin.SingleColumn("channel"),
+			Hydrate:    listConversationMembers,
+		},
+		Columns: []*plugin.Column{
+			{Name: "channel", Type: proto.ColumnType_STRING, Description: "ID of the conversation to retrieve members for."},
+			{Name: "id", Type: proto.ColumnType_STRING, Transform: transform.FromField("ID"), Description: "Unique identifier for the user."},
+		},
+	}
+}
+
+func listConversationMembers(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	api, err := connect(ctx, d)
+	if err != nil {
+		plugin.Logger(ctx).Error("slack_conversation_members.listConversationMembers", "connection_error", err)
+		return nil, err
+	}
+	channelID := d.KeyColumnQuals["channel"].GetStringValue()
+	opts := &slack.GetUsersInConversationParameters{ChannelID: channelID, Cursor: "", Limit: 1000}
```

Comment:
Can you please add a check in here if the `limit` passed into the query is less than the default limit of 1000, then `Limit` req param is adjusted? Please see https://github.com/turbot/steampipe-plugin-mongodbatlas/blob/main/mongodbatlas/table_mongodbatlas_org_event.go#L205-L208 as a reference.

Reply:
Ah I see what you mean. Wait.